examples/python/mixed/reader.py

```diff
@@ -1,38 +1,44 @@
+"""Reader without using the wait call."""
 ##############################################################################
 # Copyright (c) 2005-2015 Real-Time Innovations, Inc. All rights reserved.
 # Permission to modify and use for internal purposes granted.
 # This software is provided "as is", without warranty, express or implied.
 ##############################################################################
 
-import sys
-import os
-filepath = os.path.dirname(os.path.realpath(__file__))
-sys.path.append(filepath + "/../../../");
-import time
+from sys import path as sysPath
+from os import path as osPath
+from time import sleep
 import rticonnextdds_connector as rti
 
+filepath = osPath.dirname(osPath.realpath(__file__))
+sysPath.append(filepath + "/../../../")
 
-connector = rti.Connector("MyParticipantLibrary::Zero",filepath + "/../Mixed.xml");
-input = connector.getInput("MySubscriber::MySquareReader");
+connector = rti.Connector("MyParticipantLibrary::Zero",
+                          filepath + "/../Mixed.xml")
+inputDDS = connector.getInput("MySubscriber::MySquareReader")
 
-for i in range(1,500):
-	input.take();
-	numOfSamples = input.samples.getLength();
-	for j in range (1, numOfSamples+1):
-		if input.infos.isValid(j):
-			# There are two alternative way to access the sample...
-			# 1) get it as a dictionary and then access the field in the
-			#    standard python way:
-			sample = input.samples.getDictionary(j);
-			x = sample['x'];
-			color = sample['color'];
-			# or
-			# 2) access each single field with the connector API:
-			x = input.samples.getNumber(j, "x");
-			color = input.samples.getString(j, "color");
-			# This is how you get the size of a seqence:
-			seqLength = input.samples.getNumber(j, "aOctetSeq#");
-			print("I received a seqence with " + repr(seqLength) + "elements" );
-			# Print the sample
-			print(sample);
-	time.sleep(2);
+for i in range(1, 500):
+    inputDDS.take()
+    numOfSamples = inputDDS.samples.getLength()
+    for j in range(1, numOfSamples+1):
+        if inputDDS.infos.isValid(j):
+            """There are two alternative way to access the sample...
+            1) Get it as a dictionary and then access the field in the
+            standard python way:"""
+            sample = inputDDS.samples.getDictionary(j)
+            x = sample['x']
+            color = sample['color']
+
+            # 2) Access each single field with the connector API:
+            x = inputDDS.samples.getNumber(j, "x")
+            color = inputDDS.samples.getString(j, "color")
+
+            # This is how you get the size of a seqence:
+            seqLength = inputDDS.samples.getNumber(j, "aOctetSeq#")
+            # pylint: disable=C0325
```

Comment:
I would like to leave this out of the examples. Again: the goal is too keep the example as simple as possible. Comments should be here to explain features, not to make our lint look good. That's probably good to have in a test, but i would keep it out from the examples to avoid users confusion